spec/models/option_select_facet_spec.rb

```diff
@@ -59,31 +59,24 @@
     end
   end
 
-  describe "#close_facet?" do
-    subject { OptionSelectFacet.new(facet_data, []) }
-
-    context "small number of options" do
-      specify { expect(subject.close_facet?).to be false }
+  describe "#open_on_load?" do
+    context "when a facet is selected" do
+      subject { OptionSelectFacet.new(facet_data, "allowed-value-1") }
+      specify { expect(subject.open_on_load?).to be true }
     end
 
-    context "large number of options" do
-      let(:allowed_values) {
-        11.times.map { |i| { 'label' => "Label #{i}", 'value' => "allowed-value-#{i}" } }
-      }
+    context "when NO facet is selected" do
```

Comment:
Just to be a total pedant, would it be possible to lowercase the `NO` in the context?